backend/routers/router_users.py

```
@@ -0,0 +1,71 @@
+):
+    delete_user_id = await user_service.delete_user(user_id, db)
+    if delete_user_id is None:
+        raise HTTPException(status_code=404, detail='User not found')
+    return {"detail": "User deleted successfully"}
+
+
+@router.put("/{user_id}/{status}")
+async def update_user_status(
+    user_id: int,
+    status: bool,
+    db: AsyncSession = Depends(get_db)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)